lib/pf/services/manager/statsd.pm

```diff
@@ -32,13 +32,6 @@ sub generateConfig {
     my %tags;
     $tags{'template'}      = "$conf_dir/monitoring/statsd_config.js";
     $tags{'pid_file'}      = "$install_dir/var/run/statsd.pid";
-    $tags{'graphite_host'} = "$Config{'monitoring'}{'graphite_host'}";
```

Comment:
because that is not even used in the template. thought I might as well cleanup useless stuff at the same time